data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container.template

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Metal3 deployment service
+BindsTo=ironic-volume.service
+Requires=build-metal3-env.service wait-iptables-init.service image-customization.service build-ironic-env.service
+Wants=network-online.target ironic-httpd.service ironic-inspector.service ironic-ramdisk-logs.service
```

Comment:
nit: remove Wants of ironic-inspector, there is no direct dependency

Reply:
Done